packages/workmanager/tizen/src/job.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_JOB_INFO_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_JOB_INFO_H_
```

Comment:
It is recommended that the file name be job_info or the macro is declared as JOB.

Reply:
fixed it!